src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let result = sourceString.split(';').map((arr) => arr.split(':'));
```

Comment:
[CODE KNOWLEDGE] - Don't need to make a copy before each operation. Methods like `map` do not mutate the original array, so you can chain the calls without reassigning `result`.